packages/core/admin/admin/src/pages/AuthPage/components/Login/Form.js

```diff
@@ -0,0 +1,36 @@
+import React, { useEffect } from 'react';
+import { Form, useFormikContext, getIn } from 'formik';
+
+const FormWithFocus = props => {
+  const { isSubmitting, isValidating, errors, touched } = useFormikContext();
+
+  useEffect(() => {
+    if (isSubmitting && !isValidating) {
+      const errorNames = Object.keys(touched).reduce((prev, key) => {
```

Comment:
This is not a strong request for changes, but couldn't this be done with a filter instead of a reduce?